packages/fields/src/fields/template-title/view.tsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import type { Template } from '../../types';
+import { BaseTitleView } from '../title/view';
+
+export default function PageTitleView( { item }: { item: Template } ) {
+	return (
+		<BaseTitleView item={ item } className="fields-field__template-title" />
```

Comment:
Can't we use the `TitleView` for this one? We don't seem to use the class anywhere that I've found. If this is necessary, `TemplateTitleView` would be a more apt name.

Reply:
It was using `TitleView` before and thought you suggested every field to have a related class name. I'll revert this one.

Reply:
No, I only suggested the classes in place to have some kind of coherence :)